### Theory
<p>The protein quantitation assays can be done by UV and visible range spectroscopy. Protein quantitation involves UV absorbance and colorimetric estimation. The most reliable and consistent colorimetric assays include two categories (i) with Cu-chelating agents (ii) and Protein-dye-based chemistry. </p>
<p>Cu-chelating agents, (a) Lowry method (b) Bicinchoninic acid assay (BCA) and protein dye-based chemistry: Coomassie /Bradford</p>
<h2>Selection of the protein assay</h2>
<p>The selection of the appropriate assay analysis in perspective of the properties of the protein to be assayed by the assay system. While choosing the protein assay, the common factors to be taken into consideration are the sensitivity (lower detection limit and limitations, compatibility with the substances present (detergents, reducing agents, chelating agents, solvents) in the assay system, and the solvent of the test protein (if in liquid form), choice of matched protein for the standard curve, protein to protein variation.</p>
<h2>Ultraviolet absorbance Spectroscopy.</h2>
<p>Proteins typically show a characteristic UV absorption spectrum around 280 nm due to the presence of the aromatic amino acids tyrosine and tryptophan. If a protein's primary sequence lacks these amino acids, the UV absorption method may yield inaccurate results.</p>
<p>For precise measurements, quartz crystal cuvettes are used instead of plastic ones because plastic can leach contaminants and is not transparent to UV light. It's also crucial to avoid buffer components with strong UV absorbance, such as certain detergents like Triton X-100. "Blank" samples should be measured using the buffer solution without any protein.</p>
<p>UV absorbance is often used to estimate protein concentration. If the molar extinction coefficient of the protein is known, the Beer-Lambert law can be applied for accurate quantitation. The Beer-Lambert law is expressed as:</p>
<p>A=ϵ⋅c⋅l</p>
<ul>
    <li>A is the absorbance,</li>
    <li>ϵ is the molar extinction coefficient,</li>
    <li>c is the concentration of the analyte,</li>
    <li>l is the path length in cm.</li>
</ul>
<p>This method assumes the protein sample is pure and free from other UV-absorbing substances, such as nucleotide cofactors, heme, or iron-sulfur centers.</p>
<p>This method assumes that the protein is pure and free from UV-absorbing nonprotein components such as bound nucleotide cofactors, heme, or iron-sulfur centers.</p>
<h2>Dye-based protein assays</h2>
<p>The dye-based assay reagents can be economically prepared and in bulk and have stability.</p>
<h2>Protein concentration standards:</h2>
<p>Before starting the assay, it is important to have a precise or nearly precise standard curve to determine the unknown concentration of the protein. In an ideal situation, the standard used as the protein standard in a quantitative assay is exactly like the test protein. Practically, it is not always possible to have a matched protein as a standard, in such cases, commercially available standard proteins are used. For example, BSA, gamma globulins, immunoglobulins) The prepared standard should be made at a higher concentration and stored at -20o C.</p>
<h2>Coomassie Blue (Bradford) Protein Assay</h2>
<p>The measuring range of this assay is 1 -50 µg. The Bradford assay used G-250 dye for quantitation purposes. The binding of arginine, histidine, phenylalanine, tryptophan, and tyrosine residues. to the dye at acidic pH gives the basis for the quantitation. Upon binding protein, a metachromatic shift from 465 to 595 nm is observed due to the stabilization of the anionic form of the dye. The observed signal is due to the interaction with arginine residues, which results in the wide protein-to-protein variation characteristic of Bradford assays.</p>
<h2>Lowry (Alkaline Copper Reduction Assays) (Range: 5–100 µg)</h2>
<p>The two step Lowry assay (Lowry et al., 1951):</p>
<ol>
    <li>First, the Biuret reaction involves the reduction of copper (Cu 2+ to Cu+) by proteins in alkaline solutions,</li>
    <li>Second, the reduction of the Folin–Ciocalteu reagent (phosphomolybdate and phosphotungstate) (Peterson, 1979) produces a characteristic blue color with absorbance maxima at 750 nm.</li>
</ol>
<p>The assay displays protein sequence variation, as color development is due not only to the reduced copper–amide bond complex but also to tyrosine, tryptophan, and to a lesser extent cystine, cysteine, and histidine residues (Peterson, 1977; Wu et al., 1978).</p>
<p>The modified Lowry assay has been reduces its sensitivity to interfering agents, increase the dynamic range and increase the speed and resulting stability of the color formation (Peterson, 1979).</p>
<h2>Bicinchoninic Acid (BCA) (Range: 0.2–50 µg)</h2>
<p>The BCA reaction forms an intense purple complex with cuprous ions (Cu+) resulting from the reaction of protein and alkaline Cu2+. The residues that contribute to the reduction of Cu 2+ include the cysteine, cystine, tryptophan, tyrosine, and the peptide bonds (Smith et al., 1985).</p>
<p>The temperature is critical for the chemical reaction as it varies with the protein variability.</p>
<p>More color formation is observed due to the higher reactivity of tryptophan, tyrosine, and peptide bonds at higher temperature. The sample-to-working reagent ratio can be varied to maximize signal or reduce assay interference, typically ratios of 8–20-fold excess of BCA working reagent are added to the protein sample.</p>

